package.json

```diff
@@ -39,10 +40,18 @@
     "eslint": "^8.23.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.2.1",
+    "husky": ">=6",
+    "lint-staged": ">=10",
     "prettier": "^2.7.1",
     "prisma": "^4.3.1",
     "rimraf": "^3.0.2",
     "ts-node-dev": "^2.0.0",
     "typescript": "^4.8.2"
+  },
+  "lint-staged": {
+    "./src/**/*.ts": [
+      "eslint --cache --fix",
+      "prettier --write"
```

Comment:
From student feedback, it's better to have prettier first

Reply:
Oh, TIL. Fixed in ac67c5db48b1cb5b5f051d6a1ecc837af2dcf448